sale_order_margin_percent/views/sale_order_margin_percent_view.xml

```diff
@@ -8,6 +8,16 @@
             <field name="margin" position="after">
                 <field name="percent" widget="progressbar"/>
             </field>
+            <xpath expr="//field[@name='order_line']/form//field[@name='tax_id']" position="before">
+                <field name="percent" string="Margin" widget="progressbar"/>
+            </xpath>
+           <xpath expr="//field[@name='order_line']/tree//field[@name='price_subtotal']" position="after">
+                <field name="percent" string="Margin"/>
+                <!-- The button is used only to display % icon
+                    since is not possible to add html to tree view yet-->
+                <button icon="fa-percent"/>
```

Comment:
not sure this hack (adding a button just for a label, AFAIK) is interesting.

See invoice module : https://github.com/OCA/margin-analysis/blob/12.0/account_invoice_margin/views/account_invoice_margin_view.xml#L84